Revit_Core_Engine/Convert/Physical/FromRevit/Construction.cs

```diff
@@ -35,6 +36,24 @@ public static partial class Convert
         /****               Public Methods              ****/
         /***************************************************/
 
+        [Description("Converts a Revit HostObjAttributes to an Physical.Constructions.Construction.")]
+        [Input("hostObjAttributes", "Revit HostObjAttributes to be converted.")]
+        [Input("settings", "Revit adapter settings to be used while performing the convert.")]
```

Comment:
That would be different to the convention done by the rest of BHoM, so for consistency I would not agree.

Reply:
I have never thought of that tbh. But agreed with @FraserGreenroyd in this case, I think the general (unwritten?) convention in BHoM is not to mention that a parameter is optional.

However, maybe it would make sense to refine the `Description` query to check for the default value and mention it if true?

Reply:
OKe, good to know!